### ISSUE TITLE
Create a gatsby plugin

### ISSUE THREAD

**Issue by atrauzzi** [atrauzzi/protoculture#22]
Need a way to boot the DI system and use service providers inside of Gatsby applications.  Thought about this on the bike ride today:

 - Split `Bundle` into two modules, one that includes polyfills and reexports, and another one that is just the bundle base class, no funny business.  That way something like gatsby can still retain control of the root execution
 - The gatsby integration code somehow locates the module containing our suite subclass - helped by https://github.com/gatsbyjs/gatsby/issues/1701
 - Gatsby integration code instantiates the suite and runs it, possibly also exporting a closure'd variable so that future references are available (possibly not necessary, but might be a nice to have) - blocked by https://github.com/gatsbyjs/gatsby/issues/1715

✏️

**Comment by atrauzzi** [comment]
https://github.com/atrauzzi/gatsby-plugin-protoculture
